aws/resource_aws_glue_resource_policy_test.go

```diff
@@ -0,0 +1,174 @@
+package aws
+
+import (
+	"fmt"
+	"github.com/aws/aws-sdk-go/service/glue"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	awspolicy "github.com/jen20/awspolicyequivalence"
+	"testing"
+)
+
+func CreateTablePolicy(action string) string {
+	return fmt.Sprintf(`{
+  "Version" : "2012-10-17",
+  "Statement" : [
+    {
+      "Effect" : "Allow",
+      "Action" : [
+        "%s"
+      ],
+      "Principal" : {
+         "AWS": "*"
+       },
+      "Resource" : "arn:aws:glue:%s:%s:*"
+    }
+  ]
+}`, action, testAccGetRegion(), testAccGetAccountID())
```

Comment:
```suggestion
      "Resource" : "arn:%s:glue:%s:%s:*"
    }
  ]
}`, action, testAccGetPartition(), testAccGetRegion(), testAccGetAccountID())
```

Reply:
This change is to prevent test failures when run in GovCloud, otherwise this is good to go.

Reply:
Done

Reply:
Reran tests after the change:
```
TF_ACC=1 go test ./aws -v -count 1 -parallel 20 -run=TestAccAWSGlue_serial/ResourcePolicy -timeout 120m
=== RUN   TestAccAWSGlue_serial
=== RUN   TestAccAWSGlue_serial/ResourcePolicy
=== RUN   TestAccAWSGlue_serial/ResourcePolicy/basic
=== RUN   TestAccAWSGlue_serial/ResourcePolicy/update
=== RUN   TestAccAWSGlue_serial/ResourcePolicy/disappears
--- PASS: TestAccAWSGlue_serial (106.69s)
    --- PASS: TestAccAWSGlue_serial/ResourcePolicy (106.69s)
        --- PASS: TestAccAWSGlue_serial/ResourcePolicy/basic (31.54s)
        --- PASS: TestAccAWSGlue_serial/ResourcePolicy/update (49.40s)
        --- PASS: TestAccAWSGlue_serial/ResourcePolicy/disappears (25.74s)
PASS
ok      github.com/terraform-providers/terraform-provider-aws/aws       108.422s
```